common/ledger/blockledger/fileledger/impl.go

```diff
@@ -93,6 +94,7 @@ func (fl *FileLedger) Iterator(startPosition *ab.SeekPosition) (blockledger.Iter
 
 	iterator, err := fl.blockStore.RetrieveBlocks(startingBlockNumber)
 	if err != nil {
+		logger.Warnf("Failed to initialize block iterator from blockNum = [%d] with error = (%v)", startingBlockNumber, err)
```

Comment:
We have been preferring to use the following function for logging
```suggestion
logger.Warnw("Failed to initialize block iterator", "blockNum", startingBlockNumber, "error", err)
```